src/vma/dev/rfs.h

```diff
@@ -117,15 +117,20 @@ typedef struct __attribute__ ((packed)) ibv_flow_attr_eth_ipv4_tcp_udp {
 	vma_ibv_flow_spec_eth         eth;
 	vma_ibv_flow_spec_ipv4        ipv4;
 	vma_ibv_flow_spec_tcp_udp     tcp_udp;
+	vma_ibv_flow_spec_action_tag  flow_tag; // must be the last as struct can be used without it
```

Comment:
@olgk flow_tag is only added to Ethernet interfaces. Why not adding flow tag to IB interfaces as well?